src/main/java/com/ita/if103java/ims/mapper/UserRowMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.ita.if103java.ims.mapper;
```

Comment:
better to move jdbc mappers to separate package com.ita.if103java.ims.mapper.jdbc or any other name